src/QMCTools/ppconvert/src/CMakeLists.txt

```diff
@@ -0,0 +1,20 @@
+SUBDIRS(common)
+
+add_executable(ppconvert EXCLUDE_FROM_ALL "")
+
+target_sources(ppconvert
+  PRIVATE
+    "${CMAKE_CURRENT_SOURCE_DIR}/CubicSpline.cc"
```

Comment:
I don't think the ${CMAKE_CURRENT_SOURCE_DIR} is necessary.  Just the source file name w/o quotes should work.